ddtrace/ext/ci.py

```diff
@@ -91,6 +95,13 @@ def tags(env=None, cwd=None):
             break
 
     git_info = git.extract_git_metadata(cwd=cwd)
+    try:
```

Comment:
Shouldn't the line above also be inside the `try` block?

Reply:
`git.extract_git_metadata()` includes its own exception handling so it should be fine to leave it alone.

The reason I added a try block here was that `ci.WORKSPACE_PATH` is used in both `ci.py` and `git.py` and I had to avoid a circular import by moving the workspace path extraction (via git) to happen in `ci.py`. This was the cleanest solution I could come up with, would happily accept any other suggestions though :)